src/injection/config.js

```diff
@@ -31,6 +32,7 @@ $injector
 	.registerSingleton('AltitudeService', new AltitudeService())
 	.registerSingleton('SearchResultProviderService', new SearchResultProviderService())
 	.registerSingleton('ShareService', new ShareService())
+	.registerSingleton('UnitsService', new UnitsService())
```

Comment:
Sorry, did not see this the last time... 
Effective stateless services should be registered in "Per Request" scope

Reply:
Ah...got it!

Reply:
True, therefore I meant _**Effective** stateless services ..._
Technically it has a state, but it's just a value derived from the Config Service, there no real state needed to be kept up 😌